sqlmesh/core/model/cache.py

```diff
@@ -14,7 +14,7 @@
 
 class SqlModelCacheEntry(PydanticModel):
     model: SqlModel
-    rendered_query: t.Optional[exp.Expression] = None
+    full_depends_on: t.Set[str]
```

Comment:
quick note, i've had issues with sets in pydantic because they are unserializable in to_dict()...

Reply:
Only if it's a complex type. It's fine for primitives.